.gitignore

```diff
@@ -14,3 +14,5 @@ packs/temp-data
 static/packs/*
 types/foundry/node_modules/
 types/foundry/package-lock.json
+.DS_Store
```

Comment:
macOS spawns these files everywhere, so I think they should be excluded

Reply:
If they're excluded, they should be ignored when building.
https://github.com/foundryvtt/pf2e/pull/4816